kedro-telemetry/tests/test_plugin.py

```diff
@@ -412,6 +421,21 @@ def test_confirm_consent_yaml_dump_error(self, mocker, fake_metadata, caplog):
         )
         assert msg in caplog.messages[-1]
 
+    @mark.parametrize(
```

Comment:
The examples for testing CI tools that don't set the CI env var to be true feel a little clunky, any ideas from the reviewers?